lint.go

```diff
@@ -1466,6 +1467,32 @@ func (f *file) lintContextArgs() {
 	})
 }
 
+// deprecatedConstants keeps the mapping from a deprecated constants
+// to its replacement
+var deprecatedConstants = map[string]string{
+	"os.SEEK_SET": "io.SeekStart",
+	"os.SEEK_CUR": "io.SeekCurrent",
+	"os.SEEK_END": "io.SeekEnd",
+}
+
+// lintDeprecatedConstants checks for the use of a deprecated constant
+// and suggest a different constant in replacement
```

Comment:
s/suggest/suggests

Also comments should have a period at the end (was `lint` run on the code? ;)

Reply:
Also merge conflicts need to be resolved.